### PR TITLE
[MRG]: Mne freeview bem surfaces

### PR DESCRIPTION
**Pull request by agramfort**
one more MNE bash command converted to python

### CODE REVIEW

**Comment by agramfort** [comment]
oups 1 sec...

**Comment by agramfort** [comment]
cleaned up !

File: mne/commands/mne_freeview_bem_surfaces.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+"""View the 3-Layers BEM model using Freeview
+
+"""
+from __future__ import print_function
+
+# Authors:  Alexandre Gramfort, Ph.D.
```
Comment (mainakjas) [inline mne/commands/mne_freeview_bem_surfaces.py]: email :)

File: mne/commands/mne_freeview_bem_surfaces.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+"""View the 3-Layers BEM model using Freeview
+
+"""
+from __future__ import print_function
+
+# Authors:  Alexandre Gramfort, <alexandre.gramfort@telecom-paristech.fr>
```
Comment (teonlamont) [inline mne/commands/mne_freeview_bem_surfaces.py]: errant comma

**Comment by dgwakeman** [comment]
Out of curiosity, what's the advantage of replacing bash scripts with python scripts? I get replacing the C-code with python, but I don't see the advantage of this.

**Comment by mainakjas** [comment]
It's less error prone

**Comment by agramfort** [comment]
How do you package bash scripts ?

**Comment by agramfort** [comment]
And people can actually submit bug fixes now...

**Comment by dengemann** [comment]
@agramfort happy?

**Comment by agramfort** [comment]
Works on my machine ;)

**Comment by Eric89GXL** [comment]
-1 we should just write our own volume + surface viz :)

Otherwise LGTM

**Comment by dgwakeman** [comment]
Is there any check to see if people have FreeSurfer installed?/ whether FreeSurfer is compatible with their OS? I tried looking down the rabbit hole of get_subjects_dir, but I couldn't find one

**Comment by agramfort** [comment]
ready to go on my end

**Comment by Eric89GXL** [comment]
Thanks @agramfort

**Pull request merged**